process/superconducting_tf_coil.py

```diff
@@ -2375,147 +2393,197 @@ def superconducting_tf_case_geometry(
-        )
-        tfcoil_variables.t_conductor = np.sqrt(
-            sctfcoil_module.t_conductor_radial * sctfcoil_module.t_conductor_toroidal
-        )
+        t_conductor_radial = dr_tf_turn - 2.0e0 * dx_tf_turn_insulation
+        t_conductor_toroidal = dx_tf_turn - 2.0e0 * dx_tf_turn_insulation
+        t_conductor = np.sqrt(t_conductor_radial * t_conductor_toroidal)
 
         # Dimension of square cable space inside conduit [m]
-        sctfcoil_module.dr_tf_turn_cable_space = (
-            sctfcoil_module.t_conductor_radial - 2.0e0 * dx_tf_turn_steel
-        )
-        sctfcoil_module.dx_tf_turn_cable_space = (
-            sctfcoil_module.t_conductor_toroidal - 2.0e0 * dx_tf_turn_steel
-        )
-        sctfcoil_module.dx_tf_turn_cable_space_average = np.sqrt(
-            sctfcoil_module.dr_tf_turn_cable_space
-            * sctfcoil_module.dx_tf_turn_cable_space
+        dr_tf_turn_cable_space = t_conductor_radial - 2.0e0 * dx_tf_turn_steel
+        dx_tf_turn_cable_space = t_conductor_toroidal - 2.0e0 * dx_tf_turn_steel
+        dx_tf_turn_cable_space_average = np.sqrt(
+            dr_tf_turn_cable_space * dx_tf_turn_cable_space
         )
 
         # Cross-sectional area of cable space per turn
-        # taking account of rounded inside corners [m2]
+        # taking account of rounded inside corners [m²]
         a_tf_turn_cable_space_no_void = (
-            sctfcoil_module.dr_tf_turn_cable_space
-            * sctfcoil_module.dx_tf_turn_cable_space
-        ) - (4.0e0 - np.pi) * sctfcoil_module.rbcndut**2
+            dr_tf_turn_cable_space * dx_tf_turn_cable_space
+        ) - (4.0e0 - np.pi) * radius_tf_turn_cable_space_corners**2
 
         if a_tf_turn_cable_space_no_void <= 0.0e0:
-            if (sctfcoil_module.dr_tf_turn_cable_space < 0.0e0) or (
-                sctfcoil_module.dx_tf_turn_cable_space < 0.0e0
-            ):
+            if (dr_tf_turn_cable_space < 0.0e0) or (dx_tf_turn_cable_space < 0.0e0):
                 error_handling.fdiags[0] = a_tf_turn_cable_space_no_void
-                error_handling.fdiags[1] = sctfcoil_module.dr_tf_turn_cable_space
-                error_handling.fdiags[2] = sctfcoil_module.dx_tf_turn_cable_space
+                error_handling.fdiags[1] = dr_tf_turn_cable_space
+                error_handling.fdiags[2] = dx_tf_turn_cable_space
                 error_handling.report_error(101)
             else:
                 error_handling.fdiags[0] = a_tf_turn_cable_space_no_void
-                error_handling.fdiags[1] = sctfcoil_module.dr_tf_turn_cable_space
-                error_handling.fdiags[1] = sctfcoil_module.dx_tf_turn_cable_space
+                error_handling.fdiags[1] = dr_tf_turn_cable_space
+                error_handling.fdiags[1] = dx_tf_turn_cable_space
                 error_handling.report_error(102)
-                sctfcoil_module.rbcndut = 0.0e0
+
+                sctfcoil_module.radius_tf_turn_cable_space_corners = 0.0e0
```

Comment:
Should this actually be like this?
```suggestion
                radius_tf_turn_cable_space_corners = 0.0e0
```